Source/ActivityIndicatorView.swift

```diff
@@ -27,7 +27,7 @@ public struct ActivityIndicatorView: View {
     var type: IndicatorType
 
     @State private var isAnimating: Bool = false
-    @State private var parameter = 0.0
+    @State private var parameters = Array<Double>(repeating: 0.0, count: 8)
```

Comment:
Could you please split this in two fields with more meaningful names? It's hard to say what is going on when I see parameters[index]

Reply:
Field `parameter` was deleted. 
`parameters` means the same as `parameter`, but some indicators have a few elements and values must be stored for each element.

Reply:
I mean it can be split in two ways. (1) It is used for scaling and opacity and stuff - I think we should have separate arrays like - scalingParameters, opacityParameters. etc. Right now they are calculated through each other - like opacity is scaling divided by 2, looks in-evident to me, better not have them look connected cause they are not. (2) There are exactly 8 of them, and any constant value gives me strong sense of anxiety. Could initialisation of this array be left for later? - initialise it inside particular indicator implementation, so that the number of parameters is determined there.

Reply:
> Could initialisation of this array be left for later?
I understand you, but no, it is @state property, cannot be done initialize on execution without errors 
`Modifying state during view update, this will cause undefined behavior.`
The best way to avoid it is to implement each indicator as a separate struct. 

> we should have separate arrays like - scalingParameters, opacityParameters. 
yes you're right - it's better this way

Reply:
> The best way to avoid it is to implement each indicator as a separate struct.
Maybe that's not a bad decision. Leaving the public API untouched of course, separating them into small structs (with common ancestor if need be) with their own parameter arrays and body. Maybe we'll even manage to avoid AnyView casts this way.